.gitlab-ci.yml

```diff
@@ -101,8 +101,9 @@ cargo-check-benches:
   <<:                              *docker-cache-status
   script:
     - time (
-        cargo check --all --benches --exclude ethash --target $CARGO_TARGET --locked --verbose --color=always;
-        (cd ethash; time cargo check --benches --features bench --target $CARGO_TARGET --locked --verbose --color=always)
+        (cargo check --all --benches --exclude ethash --exclude verification --target $CARGO_TARGET --locked --verbose --color=always);
+        (pushd ethash; cargo check --benches --features bench --target $CARGO_TARGET --locked --verbose --color=always; popd);
```

Comment:
Technically you don't need pushd/popd here as the commands are run in a subshell via `(...)`.
The problem with current approach is that it will ignore the result of the all but the last command, so e.g. if `cargo check --benches --exclude ethash --exclude verification` fails, but ethcore/verification succeeds, this check won't fail.
I think we should run each command as a separate step (`-`).

Reply:
> I think we should run each command as a separate step (-).

Good point. Doing that.